### PR TITLE
Add initial cosmos testing

### LINKED ISSUE #9: Bootstrap testing/CICD
- [x] Set up [pytest](https://docs.pytest.org/en/latest/)
- [x] Set up [cosmosdb-server](https://github.com/zeit/cosmosdb-server) to test cosmos functionality
- [x] Set up GitHub Actions CI

### PR DESCRIPTION
**Pull request by ridhoq**
Using my own [Dockerfile](https://github.com/ridhoq/cosmosdb-server/blob/docker/Dockerfile) for [zeit/cosmosdb-server](https://github.com/zeit/cosmosdb-server) for integration testing cosmos interactions

Closes #9

### CODE REVIEW

File: ditto-markov/ditto_markov/config.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import os
+from distutils.util import strtobool
 
 COSMOS_ACCOUNT_URI = os.getenv('COSMOS_ACCOUNT_URI')
 COSMOS_ACCOUNT_KEY = os.getenv('COSMOS_ACCOUNT_KEY')
 COSMOS_DATABASE_ID = os.getenv('COSMOS_DATABASE_ID')
 AZURE_STORAGE_CONNECTION_STRING = os.getenv('AZURE_STORAGE_CONNECTION_STRING')
+COSMOS_DISABLE_TLS = bool(strtobool(os.getenv('COSMOS_DISABLE_TLS') or "False"))
```
Comment (austinpray) [inline ditto-markov/ditto_markov/config.py:8]: TIL `strtobool`

File: ditto-markov/docker-compose.yaml [hunk 2]
```diff
@@ -11,4 +11,29 @@ services:
...
+    build:
+      context: .
+      target: development
+    image: ridhoq/ditto/ditto-markov/test
+    environment:
+      COSMOS_ACCOUNT_URI: https://cosmosdb:3000
+      COSMOS_DATABASE_ID: ditto-test
+      COSMOS_DISABLE_TLS: "True"
+    volumes:
+      - ./pytest.ini:/app/pytest.ini
```
Comment (austinpray) [inline ditto-markov/docker-compose.yaml:34]: what was this mount for?

File: ditto-markov/pytest.ini [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+testpaths = tests
+filterwarnings = ignore:Unverified HTTPS request
```
Comment (austinpray) [inline ditto-markov/pytest.ini:3]: based

**Review by austinpray — APPROVED** [review]
LGTM 
image

File: ditto-markov/docker-compose.yaml (hunk 2 above)
Comment (ridhoq) [reply ditto-markov/docker-compose.yaml:34]: I was tryna get warning suppression to work but we don't really need it to be mounted. I'll remove that in the next PR 👍

**Pull request merged**